app/pages/Settings/index.js

```diff
@@ -314,15 +314,6 @@ export default class Settings extends Component {
         { this.renderContent() }
         <Switch>
           <Route path="/settings/wallet/account-index" component={AccountIndexModal} />
-          <Route
```

Comment:
It looks like something changed about the way you're working with routes in settings - I'm getting a warning in console every time I execute that code path in Settings.  Details below:

![slash-account](https://user-images.githubusercontent.com/8230144/96225314-c2c85100-0f45-11eb-8c64-7f0bdc912960.gif)